src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml

```diff
@@ -292,154 +405,146 @@
-                <Image.ToolTip>
-                    <TextBlock Text="{Binding Path=AddSSHConnectionCommand.ToolTip, Mode=OneWay}" />
-                </Image.ToolTip>
-            </vsimaging:CrispImage>
-        </Button>
+                IsEnabled="{Binding AddSSHConnectionCommand.CanExecute}"
+                Content="{x:Static local:UIResources.AddNewSSHConnectionButtonLabel}"
+                ToolTip="{x:Static local:UIResources.AddNewSSHConnectionToolTip}"/>
 
-        <!-- Advanced Settings -->
-        <Expander x:Name="advancedSettingsExpander"
-                  x:Uid="uidadvancedSettingsExpander"
-                  HorizontalAlignment="Stretch"
-                  Header="{x:Static local:UIResources.AdvancedSettingsHeader}"
-                  Grid.Row="2"
-                  Grid.ColumnSpan="2"
-                  Margin="0,10,0,0"
-                  VerticalAlignment="Top"
-                  ExpandDirection="Down">
-            <Grid>
-                <Grid.RowDefinitions>
-                    <RowDefinition Height="Auto" />
-                </Grid.RowDefinitions>
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="Auto" />
-                    <ColumnDefinition Width="*" />
-                </Grid.ColumnDefinitions>
-                
-                <Label x:Uid="uidHostnameLabel"
-                       Grid.Row="0"
-                       Grid.Column="0"
-                       x:Name="Hostname"
-                       Target="{Binding ElementName=hostnameTextBox}"
-                       Margin="19,5,0,5"
-                       Content="{x:Static local:UIResources.HostnameLabel}"
-                       ToolTip="{x:Static local:UIResources.HostnameTip}" />
+        <!-- Hostname label and textbox -->
 
-                <TextBox x:Uid="uidHostnameTextBox"
-                         Grid.Row="0"
-                         Grid.Column="1"
-                         x:Name="hostnameTextBox"
-                         Text="{Binding Path=Hostname, Mode=TwoWay,UpdateSourceTrigger=PropertyChanged}"
-                         PreviewKeyDown="HostnameTextbox_PreviewKeyDown"
-                         Margin="0,5,0,5"
-                         Padding="2" />
-                </Grid>
-        </Expander>
+        <Label Grid.Row="2"
+               Grid.ColumnSpan="2"
+               x:Uid="uidHostnameLabel"
+               x:Name="HostnameLabel"
+               Margin="0,9,0,0"
+               Padding="0"
+               Height="16"
+               VerticalAlignment="Center"
+               Target="{Binding ElementName=hostnameTextBox}"
+               Content="{x:Static local:UIResources.HostnameLabel}" />
+        <TextBox Grid.Row="3"
+                 Grid.ColumnSpan="2"
+                 Margin="0,4,0,0"
+                 Padding="2"
+                 x:Uid="uidHostnameTextBox"
+                 x:Name="hostnameTextBox"
+                 AutomationProperties.Name="DockerHostnameOptional"
```

Comment:
You can have spaces in automation properties. Also, you probably want to have it be "Optional Docker Hostname"

Reply:
Also, don't you need to have this in a resource file for localization?